content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md

```diff
@@ -31,3 +31,12 @@ part of the Rust Gamedev Working Group, and the team lead of the Rust CTCFT
 team. I’m also one of the core developers on Veloren, an open source game
 similar to Minecraft. In the Rust communty, I run a monthly game dev meetup, and
 co-edit the monthly game dev newsletter.
+
+### Steven Kim ([@kyonghokim93](https://github.com/kyonghokim93))
+
+Hi, I’m Steven!
+
+I’m a CarletonU alumni with MA in Applied Linguistics who is now a CompSci student at Algonquin College. I am still new to many CompSci concepts, and I am excited to learn as much as I can.
```

Comment:
Super cool! There is a ton of research in CS about natural language processing, definitely check this out in your upper years!